pom.xml

```diff
@@ -59,13 +75,17 @@
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.glassfish.jersey.media</groupId>
+      <artifactId>jersey-media-json-jackson</artifactId>
```

Comment:
why this dependency?

Reply:
For some reason the dependency is not picked from the main library, bitbucket-client, and without it I am getting Exception :+1:
`org.glassfish.jersey.message.internal.MessageBodyProviderNotFoundException: MessageBodyReader not found for media type=application/json`

Reply:
so please <scope>runtime</scope> for this case

Reply:
Scope runtime added